CHANGELOG.md

```diff
@@ -4,6 +4,42 @@
 
 [1]: https://www.npmjs.com/package/nodejs-pubsub?activeTab=versions
 
+## v0.25.0
+
+02-20-2019 10:35 PST
+
+### New Features
+- refactor(ts): improve TypeScript types ([#482](https://github.com/googleapis/nodejs-pubsub/pull/482))
+- refactor(typescript): noImplicityAny for snapshot.ts and publisher.ts ([#457](https://github.com/googleapis/nodejs-pubsub/pull/457))
+
+### Bug Fixes
+- fix: ignore messages that come in after close ([#485](https://github.com/googleapis/nodejs-pubsub/pull/485))
+- fix: throw on invalid credentials and update retry config ([#476](https://github.com/googleapis/nodejs-pubsub/pull/476))
```

Comment:
I am digging into this particular change. it seems odd that so many things changed from non-idempotent to idempotent. If we can hold this for just a bit that would be super.

Reply:
@jadekler do you have input here? I want to say I remember you telling us this change was coming (apologies if I'm misremembering).